src/context/LikeVideoContext.js

```diff
@@ -31,10 +33,13 @@ const LikeVideoProvider = ({ children }) => {
                 data: { video: video },
             });
             if (response.status === 201) {
-                setLikeVideo(response.data.likes);
+              setLikeVideo(response.data.likes);
+              
+                Toast({ type: "success", msg: "You like this videos" });
             }
         } catch (error) {
-            console.log(error);
+          console.log(error);
```

Comment:
Console all your errors in console.error instead of console.log then your errors would be differentiated from the other consoles.